forgerock-auth/src/test/java/org/forgerock/android/auth/FRAuthMockTest.java

```diff
@@ -214,4 +218,51 @@ public void startTest() {
 
     }
 
+    @Test
+    public void startTestWithOptions() {
```

Comment:
Can we include a test with multiple starts for different configurations?

Reply:
Sure , we can back fill more tests for configuration

Reply:
Done